opal/Makefile.am

```diff
@@ -78,18 +78,14 @@ lib@OPAL_LIB_NAME@_la_LIBADD = \
         mca/base/libmca_base.la \
         util/libopalutil.la \
 	$(MCA_opal_FRAMEWORK_LIBS) \
-	$(opal_libevent_LIBS) \
-	$(opal_hwloc_LIBS) \
-	$(opal_pmix_LIBS)
+        $(OPAL_WRAPPER_EXTRA_LIBS)
```

Comment:
nack; we intentionally never have the libraries we build depend on the WRAPPER / PKG-CONFIG flags.  We need to fix whatever's wrong with opal_*_LIBS (and opal_*_LDFLAGS, below).  This is not an acceptable fix.  Without a clearly example of what is breaking, it is hard to suggest the correct fix.